pyproject.toml

```diff
@@ -21,7 +21,7 @@ docutils = "^0.16"
 importlib-metadata = {version = "^3.10.1", python = "<3.8"}
 python = "^3.7"
 requests = "^2.25.1"
-Sphinx = "^3.5.4"
+Sphinx = "^3.5.4 || ^4.0.0"
```

Comment:
I'd be tempted to actually set `="*"`, we'll remove version if there are know incompatibilities. You can do the same with requests.

Reply:
Great! Just updated and executed `poetry lock --no-update`